armi/reactor/blueprints/tests/test_materialModifications.py

```diff
@@ -54,13 +54,41 @@ class TestMaterialModifications(unittest.TestCase):
         xs types: [A]
 """
 
-    boronInput = uZrInput.replace("UZr", "B")
+    b4cInput = r"""
+nuclide flags:
+    B: {burn: false, xs: true}
+    C: {burn: false, xs: true}
+blocks:
+    poison: &block_poison
+        poison:
+            shape: Hexagon
+            material: B4C
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+assemblies:
+    assem a: &assembly_a
+        specifier: IC
+        blocks: [*block_poison]
+        height: [1.0]
+        axial mesh points: [1]
+        xs types: [A]
+"""
 
     def loadUZrAssembly(self, materialModifications):
-        yamlString = self.uZrInput + "\n" + materialModifications
+        return self._loadAssembly(self.uZrInput, materialModifications, "fuel a")
+
+    @staticmethod
+    def _loadAssembly(bpBase: str, materialModifications: str, assem: str):
+        yamlString = bpBase + "\n" + materialModifications
         design = blueprints.Blueprints.load(yamlString)
         design._prepConstruction(settings.Settings())
-        return design.assemblies["fuel a"]
+        return design.assemblies[assem]
+
+    def loadB4CAssembly(self, materialModifications: str):
```

Comment:
I'm not convinced this method should exist. It's a one-line method, that just calls another method, and it is only used once.